custom-pages/universal-login/src/components/Register/Register.js

```diff
@@ -215,14 +234,21 @@ class Register extends Component {
                   aria-labelledby="error-summary-title"
                 >
                   <p className="lead">There is a problem</p>
-                  <button
-                    role="link"
-                    tabIndex="0"
-                    onKeyPress={this.goToAlert}
-                    onClick={this.goToAlert}
-                  >
-                    Click here to see the errors
-                  </button>
+
+                  <ul>                    
+                    {Object.keys(errors).map((errorName, idx) => {
+                      if (errors[errorName]) {
+                        return (
+                          <li key={idx}>
+                            <a href={`#${errorName}`} onClick={(e) => this.goToAlert(errorName, e)}>
```

Comment:
i'm wondering if this should have an aria-label on each link telling you that clicking on the link (which just has the text of an error message) actually takes you to the field in question that's thrown the error. what do you think?

Reply:
Yeah, completely agree. Will sort.